test/OpenTelemetry.Exporter.Jaeger.Tests/JaegerExporterTests.cs

```diff
@@ -14,8 +14,10 @@
 // limitations under the License.
 // </copyright>
 
+using System;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Linq;
```

Comment:
nit - it seems these namespace got added without any code actually using them?

Reply:
thank you for reviewing!